comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Comment(TimestampMixin, models.Model):
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    recipe = models.ForeignKey('recipes.Recipe', on_delete=models.CASCADE)
+    text = models.TextField(max_length=2000)
```

Comment:
`max_length` on a `models.TextField` only changes how big the forms textfield box is when displayed. It does not enforce validation of that max_length at the form validation, model validation, or database level. Just fyi